fastestimator/estimator.py

```diff
@@ -155,17 +157,21 @@ def _prepare_traces(self, run_modes: Set[str]) -> None:
             extra_monitor_keys.update(trace.fe_monitor_names - trace_outputs)
         # Add the essential traces
         if "train" in run_modes:
-            self.traces_in_use.insert(0, TrainEssential(monitor_names=self.monitor_names.union(extra_monitor_keys)))
+            self.traces_in_use.insert(0, TrainEssential(
+                monitor_names=self.monitor_names.union(extra_monitor_keys)))
             no_save_warning = True
             for trace in get_current_items(self.traces_in_use, run_modes=run_modes):
                 if isinstance(trace, (ModelSaver, BestModelSaver)):
                     no_save_warning = False
             if no_save_warning:
-                print("FastEstimator-Warn: No ModelSaver Trace detected. Models will not be saved.")
+                print(
```

Comment:
it looks like the formatter settings may not be configured correctly. there's no reason for most of these format changes